src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -1091,6 +1093,31 @@ export class BaseEditCourseForm extends React.Component {
               disabled={disabled}
               optional
             />
+            {administrator && (
+              <Field
+                name="watchers_list"
+                component={ReduxFormCreatableSelect}
+                label={(
+                  <FieldLabel
+                    id="watchers.label"
+                    text="Watchers"
+                    helpText={(
+                      <p>
+                        A list of email addresses that will receive
+                        notifications when the course run of the course is published or reviewed.
+                      </p>
+                    )}
+                    optional
+                  />
+                )}
+                isMulti
+                disabled={!(courseInfo?.data?.course_run_statuses?.includes(REVIEW_BY_INTERNAL) && administrator)}
```

Comment:
- If I remember the spec, the watchers can be added at any point (but not once scheduled/published). Double-check this.
- We should make this field prominent, either moving to the side pane or moving higher in the course form. If the side pane does not take much effort, that would be preferrable. This particular field does not impact the course content behavior.

Reply:
Also, the form remains in an uncleaned state due to the ordering of watcher fields coming from backend. The form field needs to be refreshed. It is already happening for many of the fields in Publisher.

Reply:
1. I've doubled check this from specs doc
![image](https://github.com/openedx/frontend-app-publisher/assets/78806673/66a1a474-229c-4932-b4b4-711e26f19f51)
2. Ok, I'm moving this field upwards after slug field to make it prominent, adding a side pane and manage field state require some effort, can be done in some followup PR
3. Working on it

Reply:
> Also, the form remains in an uncleaned state due to the ordering of watcher fields coming from backend. The form field needs to be refreshed. It is already happening for many of the fields in Publisher.
Done